test/components/advanced-options.test.tsx

```diff
@@ -16,71 +17,55 @@ describe("AdvancedOptions component", () => {
     };
   });
 
-  it("respects `sidebar` config option and renders only chosen components", () => {
-    // Case 1
-    const allOptions = [
-      ["interactions", "Interaction"],
-      ["timestep", "Model Speed"],
-      ["latLongLines", "Latitude and Longitude Lines"],
-      ["plateLabels", "Plate Labels"],
-      ["velocityArrows", "Velocity Arrows"],
-      ["forceArrows", "Force Arrows"],
-      ["eulerPoles", "Euler Poles"],
-      ["boundaries", "Plate Boundaries"],
-      ["wireframe", "Wireframe"],
-      ["save", "Share Model"]
-    ];
+  const allOptions = [
+    ["interactions", "Interaction"],
+    ["timestep", "Model Speed"],
+    ["latLongLines", "Latitude and Longitude Lines"],
+    ["plateLabels", "Plate Labels"],
+    ["velocityArrows", "Velocity Arrows"],
+    ["forceArrows", "Force Arrows"],
+    ["eulerPoles", "Euler Poles"],
+    ["boundaries", "Plate Boundaries"],
+    ["wireframe", "Wireframe"],
+    ["save", "Share Model"]
+  ];
+
+  it("respects `sidebar` config option and renders only chosen components - case 1", () => {
     config.sidebar = allOptions.map(e => e[0]);
-    let wrapper = mount(
-      <AdvancedOptions simulationStore={store} />
-    );
+    render(<AdvancedOptions simulationStore={store} />);
     allOptions.map(e => e[1]).forEach(label => {
-      expect(wrapper.text()).toEqual(expect.stringContaining(label));
+      expect(screen.queryByText(label)).toBeInTheDocument();
```

Comment:
For this usage (and others down below) I believe `getByText()` would be more conventional. `queryByText()` is generally used when zero or more than one instance are expected.

Reply:
Done.